tests/pq/test_exec.py

```diff
@@ -144,3 +157,22 @@ def test_describe_portal(pgconn):
     pgconn.finish()
     with pytest.raises(psycopg.OperationalError):
         pgconn.describe_portal(b"cur")
+
+
+@pytest.mark.crdb_skip("server-side cursor")
+@pytest.mark.libpq(">= 17")
```

Comment:
Like for other version-depending tests, I would like to see a test `@pytest.mark.libpq("< 17")`.

Reply:
Okay I added those tests reverse version depending tests too, and also added CI jobs that test with latest development version of libpq.